common/views/components/SearchBar/SearchBar.tsx

```diff
@@ -42,6 +42,7 @@ const SearchBar: FunctionComponent<{ type: string }> = ({ type }) => {
   return (
     <form
       role="search"
+      id='searchPageForm'
```

Comment:
I can't quite remember how the linting setup works these days (might be a precommit hook that needs to be set up?), and it should be run in CI regardless of what you do locally, but this should be double quotes to match other stuff.

cc @wellcomecollection/js-ts-reviewers anyone know why this isn't getting caught anywhere?

Reply:
I don't think it's caught by our linter; I did a big commit to go back and fix them all up, but I couldn't work out how to have the lint enforce them going forward.

Reply:
Just checked and it already is enforced/caught by the linter; now I look at it the problem is indeed that we don't lint in CI any more and I think the precommit hook situation is questionable too 😱. I'll take a look tomorrow.

Have also just realised that this is documented!
https://github.com/wellcomecollection/wellcomecollection.org/blob/21f6d0ef81f3f431c7528ad1c073993014f6988c/README.md#L150-L157